src/mnode/rpc/masternode.cpp

```diff
@@ -115,6 +115,10 @@ Available modes:
         masterNodeCtrl.masternodeManager.UpdateLastPaid(pindex);
     }
 
+    auto is_filtered = [&](const auto&... filters) -> bool {
```

Comment:
As I understand, the meaning of this lambda is the opposite: is_not_filtered - i.e. to check that strFilter is not in filters. If strFilter is defined - only items that pass this filter (that have substring) will be added to the output.

Reply:
also can make this lambda const

Reply:
The context that the lambda is being used is asking the question if the current item is filtered and if so, continuing to the next item.  I could change the conditionals to be something like `if (!is_not_filter(...)) continue;` but that's a double negative.

Sure, I can make it const.

Reply:
I think the parameter name is what causes confusion.  `strFilter` is really a search term not a filter term.  We are using that variable to find the rows we want as opposed to the ros we don't want.